remote/server.go

```diff
@@ -241,6 +241,73 @@ func handleNetworks(ctx context.Context, sm subnet.Manager, w http.ResponseWrite
 	jsonResponse(w, http.StatusOK, wr)
 }
 
+// POST /{network}/reservations
+func handleAddReservation(ctx context.Context, sm subnet.Manager, w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
```

Comment:
The handler does not need to close the request body. The http server will do that for you.

Refer: https://golang.org/pkg/net/http/#Request